run.sh

```diff
@@ -27,7 +27,12 @@ runcommand() {
 }
 
 # create passwd entry for current uid, fix HOME variable
-create_user $(id -u)
+# only execute, if the current uid does not exist.
+if id $(id -u) >/dev/null 2>/dev/null; then
+    echo Not creating user with uid $(id -u) because he already exists
```

Comment:
I don't want to be picky, but could you rewrite this message in a gender neutral form, please?